stablehlo/dialect/VhloOps.cpp

```diff
@@ -365,5 +367,28 @@ LogicalResult SelectAndScatterOpV1::validateConstraint(mlir::Operation* op,
   return verifyConstraint_0_17_0(op, targetVersion);
 }
 
+LogicalResult CustomCallOpV1::validateConstraint(mlir::Operation* op,
+                                                 Version targetVersion) {
+  if (targetVersion < Version(0, 20, 0)) return success();
```

Comment:
Uber-nit: consider adding a small helper `isVersionBeforeSymbolRefV1AttrIntroduced` (ideally, something less verbose), or creating an appropriately-named global constant for `Version(0, 20, 0)` to avoid having a magic constant duplicated in a few places.

Reply:
More generally, I think it would be a good idea to have some place where minor versions are explicitly tagged with a short description of the relevant changes. WDYT?